sympy/core/containers.py

```diff
@@ -50,6 +50,12 @@ class Tuple(Basic):
     def __new__(cls, *args, **kwargs):
```

Comment:
Probably this should use an explicit keyword argument:
```python
def __new__(cls, *args, sympify=False):
```

Reply:
Would it be confusing if this gets changed in this PR? Since it solves a different issue. Note also that the current default value is `sympify=True`.